README.md

```diff
@@ -227,9 +227,9 @@ If you configure your installation to have the React SDK static content served f
 
   * https://\<**host name or IP address of React SDK server**>:<**port you’re using**>/auth.html (for the portal use case)
```

Comment:
(for all embedded use cases and for any full re-authentication challenges for non-embedded use cases)